include/lsst/daf/base/PropertySet.h

```diff
@@ -58,19 +58,14 @@
 
 namespace lsst {
 namespace daf {
-
-namespace persistence {
-class PropertySetFormatter;
-}  // namespace persistence
-
 namespace base {
 
 #if defined(__ICC)
 #pragma warning(push)
 #pragma warning(disable : 444)
 #endif
 
-class LSST_EXPORT PropertySet : public Persistable, public Citizen {
+class LSST_EXPORT PropertySet : public Citizen {
```

Comment:
should you remove `#include "lsst/daf/base/Persistable.h"`?

Reply:
Unfortunately, no - it's still used in some accessors (a `PropertySet` is no longer a `Persistable` itself, but it can still store `Persistables` and probably will need to until we can drop pex_policy completely).